### DISCUSSION TITLE
Official Public RPC serverinfo

### DISCUSSION THREAD

**Discussion by vasuxdev** [dogecoin/dogecoin#3510] [General]
Do we have any rpc servers hosted by dogecoin foundation ?
If not can we host an rpc endpoints in the domain dogecoin.com ?It would be nice if we have some nodes from dogecoin.com since this domain is trustworthy

**Comment by patricklodder** [comment]
It's not secure to expose a dogecoind RPC to the public, nor is it safe for you to trust anyone's RPC.

You can run your own though, it's easy. See [the getting started guide](https://github.com/dogecoin/dogecoin/blob/master/doc/getting-started.md)
